sdks/java/io/neo4j/src/main/java/org/apache/beam/sdk/io/neo4j/Neo4jIO.java

```diff
@@ -1126,11 +1155,11 @@ private WriteUnwindFn(
     public void processElement(ProcessContext context) {
       // Map the input data to the parameters map...
       //
-      ParameterT parameters = context.element();
+      KV<Integer, ParameterT> parameters = context.element();
```

Comment:
It still looks a bit unuaual to me

- A `Reshuffle.AssignShardFn(parallelism)` is used to map Element -> KV<Integer, Elemenr>, but in the downstream it just takes value, within same fused stage
- The DoFn output type changes from Void to ParameterT, but WriteUnwindFn still does not output anything

i.e. both change modifies the API but still no-op. Are they intended? WIll there be follow up tasks? CC: @bvolpato @fbiville